cartridges/int_alma/cartridge/templates/default/checkout/billing/paymentOptions/almaIncludes.isml

```diff
@@ -58,15 +58,26 @@
 
             var paymentData = getPaymentData(data, installments_count, deferred_days)
 
-            fragments.createPaymentForm(paymentData, {
+            const paymentForm = fragments.createPaymentForm(paymentData, {
+                showPayButton: false,
                 onSuccess: function (returnedData) {
                     window.location = returnedData.return_url;
                 },
                 onFailure: function (returnedData) {
                 },
                 onPopupClose: function (returnedData) {
                 },
-            }).mount(document.getElementById(container));
+            })
+
+            paymentForm.mount(document.getElementById(container));
+
+            document
+                .querySelector('.submit-payment')
+                .addEventListener('click', async () => {
+                    // Trigger Alma's payment process
+                    console.log('Payement with new button');
```

Comment:
remove console log and comments here please

Reply:
It's done.